src/Aliencube.AzureFunctions.Extensions.OpenApi.CLI/Program.cs

```diff
@@ -66,9 +66,9 @@ public static void Main(string[] args)
             {
                 pi = new ProjectInfo(project, configuration, target);
             }
-            catch
+            catch (Exception ex)
```

Comment:
You can just remove the try/catch if you don't intend to handle the exception.

If you did want to handle the exception in some manner, you would just "throw" rather than "throw ex" after doing so, otherwise you will lose the stack.